js/ui/scheduler/rendering_strategies/ui.scheduler.appointments.strategy.base.js

```diff
@@ -466,17 +466,15 @@ class BaseRenderingStrategy {
         return startDate;
     }
 
-    endDate(appointment, position, isRecurring) {
-        var endDate = this.instance._getEndDate(appointment),
+    endDate(appointment, position, isRecurring, ignoreViewDates = false) {
+        let endDate = this.instance._getEndDate(appointment, ignoreViewDates),
             realStartDate = this.startDate(appointment, true),
             viewStartDate = this.startDate(appointment, false, position);
```

Comment:
realStartDate and viewStartDate are const
fullDuration is let

Reply:
fullDuration is changed in this method. So, it is needed as let